src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -286,13 +286,12 @@ <h4 *ngIf="isTestRun">
                                             </button>
                                             <ng-template #continueButton>
                                                 <button
-                                                    *ngIf="submission !== null && submission.latestResult !== null && submission.latestResult !== undefined"
+                                                    *ngIf="submission && submission.latestResult && correctionRound === 0"
```

Comment:
this could be shortened with:

```suggestion
                                                    *ngIf="submission?.latestResult && correctionRound === 0"
```

Reply:
👍